docs/how-tos.rst

```diff
@@ -0,0 +1,91 @@
+**********************
+How-Tos, Tips & Tricks
+**********************
+
+This chapter contains some practical examples of using ``shiv`` in the real world,
+to give you a better idea how it can fit into your projects
+and their deployment workflows.
+
+
+Shipping Dependency Sets as a Single File
```

Comment:
What do you think about referring to these as "frozen environments" rather than "dependency sets". This is usually how we refer to them internally, but I want to know if it makes sense in the broader Python community.